Source/WebCore/css/StyleRule.h

```diff
@@ -52,6 +52,15 @@ class StyleSheetContents;
 using CascadeLayerName = Vector<AtomString>;
     
 DECLARE_ALLOCATOR_WITH_HEAP_IDENTIFIER(StyleRuleBase);
+
+// Base class to be derived by concrete style rule implementations.
+// NOTE: StyleRuleBase doesn't have (or need) a virtual destructor, as it defines
+// a custom "destroying delete" operator, which calls  the appropriate destructor for
```

Comment:
Nit: two spaces between "calls" and "the appropriate"

Reply:
Ouch, I'll fix that once the checker confirms this indeed suppresses the warning